### ISSUE TITLE
Force values to be strings

### ISSUE THREAD

**Issue by valepu** [zeke/npe#22]
Hi,
I don't know if i'm doing something wrong or it's a bug. 
I wanted to test this library to see if it was fit for my purpose and i changed the version of a package like this
`npe dependencies.package 123`
I just typed 123 as test but when i checked the `package.json` i saw it was inserted as a number (without quotes) rather than a string. Despite not being a common use case (most versions are in the form `1.2.3` or similar) this doesn't seem correct to me, it might also affect other properties if there's, for some reason, a package with a numeric value as description

I even tried concatenating the number to an empty string (it works if i concatenate to a space, but then i have `" 123"` in the `package.json`), using `"123"` and even `$( printf '%d' 123 )` (which was suggested on StackOverflow as a way to force a value to be a string) but nothing worked.

I don't think there's a case in the package.json where you can use a pure number as value, they should all be strings

**Comment by zeke** [comment]
Hi there. There does exist some logic in the code to change `true` and `false` from strings to booleans, but I don't recall that any intentional coercion of numbers is happening.

I also can't think of a case where a package.json file needs a number as a value. 🤔 

I'd be open to a PR that adds support for this if you think there's a valid reason to add it, and you're willing to add some tests for the new functionality.

Thanks!

**Comment by valepu** [comment]
Yeah i agree It's an extremely unlikely occurrence (the only example i could think of was if somebody used a number as description for some reason), I only discovered it by chance because i ran `npe dependencies.package 123`for testing purposes, i don't even need this feature just saw a bug and wanted to report it :) .  

But...as soon as i have some spare time i might look into it! Thanks
